draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -671,151 +629,104 @@ This approach is the simplest solution, as it requires no interpretation by the
 
 ~~~~ 
- +--rw vendor-specific-config-attribute-list* [conf-attribute-id]
+ +--rw opaque-config-attribute-list* [conf-attribute-id]
      +--rw conf-attribute-id             uint32
      +--rw conf-attribute-name           string
      +--rw value                         decimal64
      +--rw memory-map                    decimal64
 
-<vendor-specific-config-attribute-list xmlns="urn:example:cc">
+<opaque-config-attribute-list xmlns="urn:example:cc">
   <config-attribute-id> 1 </config-attribute-id >
   <config-attribute-name> AA </config-attribute-name>
   <value> 100 </value>
   <memory-map> 1100 </memory-map>
-</vendor-specific-config-attribute-list>
+</opaque-config-attribute-list>
 ~~~~
-{: #solution-1 title="Solution-1 for support of vendor proprietary config attributes"}
+{: #solution-1 title="Solution-1 for support of opaque config attributes"}
 
-### Vendor-Specific Configuration Attributes (Solution-2)
+### Support of Opaque Configuration Attributes (Solution-2)
 
 This approach is similar to {{solution-1}} but requires the host platform to implement lookup logic to determine the memory-map location on the pluggables. In this solution, the coherent pluggable YANG data model is augmented with the following new attributes, as shown in {{solution-2}}:
 
 * ID of new proprietary configuration attribute
 * Name of new proprietary configuration attribute
 * Value of new proprietary attribute
 
-The operator does not have visibility into the specific memory-map locations for these attributes on the coherent pluggable device. Instead, the memory-map for each new attribute is provided in the Coherent Pluggable Manifest. In this scenario, the host platform must search the pluggable manifest to locate the corresponding memory-map location for each new attribute. These values are then communicated to the pluggable via the CMIS protocol for configuration. As in {{solution-1}}, the host platform does not need to understand the syntax or semantics of the new attributes; it only needs to search the pluggable manifest to identify the memory-map locations for the new attributes.
+The operator does not have visibility into the specific memory-map locations for these attributes on the coherent pluggable device. Instead, the memory-map for each new attribute is provided in the Coherent Pluggable Repository. In this scenario, the host platform must search the pluggable Repository to locate the corresponding memory-map location for each new attribute. These values are then communicated to the pluggable via the CMIS protocol for configuration. As in {{solution-1}}, the host platform does not need to understand the syntax or semantics of the new attributes; it only needs to search the pluggable Repository to identify the memory-map locations for the new attributes.
```

Comment:
This is not what a standard is supposed to cover. Vendors are free to implement their proprietary extensions.memory-map programming is not what a standard should facilitate.